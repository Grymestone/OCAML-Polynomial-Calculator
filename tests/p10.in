(2a^2)^2
