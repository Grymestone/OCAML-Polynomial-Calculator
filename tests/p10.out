((((2*(a)^2))^2+((2*(a)^9))^2)+((5*(a)^1))^2)
4x^4 + 4x^18 + 25x^2